src/DssSpell.t.sol

```diff
@@ -36,6 +36,14 @@ interface BridgeLike {
     function l2TeleportGateway() external view returns (address);
 }
 
+interface VatLike {
+    function wards(address) external view returns (uint256);
+}
+
+interface CatLike {
+    function wards(address) external view returns (uint256);
+}
+
```

Comment:
Both `RwaInputConduitLike` and `RwaLiquidationLike` are not used, so they shoudl be removed.

Reply:
adjusted